### ISSUE TITLE
Incorrect root path

### ISSUE THREAD

**Issue by lackneets** [manogi/october-mediathumb#6]
Generated image URL start with `/storage/app/media` when my october is located under sub-directory.

note: The official [MediaLibrary](https://github.com/octobercms/october/blob/master/modules/cms/classes/MediaLibrary.php#L411) generated URL is correct 

``` php
MediaLibrary::instance()->getPathUrl($path)
```

This works for me, but stupid

``` php
(object)[
    'url' => $file->publicUrl,
    'path' => $file->path,
    'thubmnail' => dirname(dirname(dirname(MediaLibrary::instance()->getPathUrl('')))) . getMediathumb($file->path, 'auto', '200')
];
```

**Comment by bastihilger** [comment]
I am now using MediaLibrary for outputting the path. I updated the master branch and already updated the plugin. Please check if it works for you now.

**Comment by lackneets** [comment]
Bug fixed, thanks

**Comment by lackneets** [comment]
Problem is back at current version, since I update October from 318 to 396
not sure if it is affected by newer plugin or october itself 

My stupid fix for now:

```php
    public function getThumbnail($img, $mode='auto', $size=250, $quality=null){
        if(function_exists('getMediathumb')){
            return dirname(dirname(dirname(MediaLibrary::instance()->getPathUrl('')))) . getMediathumb($img, $mode, $size, $quality);
        }else{
            return MediaLibrary::instance()->getPathUrl($img);
        }
    }
```

FYI.

**Comment by bastihilger** [comment]
This is fixed in the new update 0.4.3
